.changeset/large-dolphins-roll.md

```diff
@@ -0,0 +1,24 @@
+---
+'@astrojs/markdown-remark': minor
+'astro': minor
+---
+
+Adds support for `excludeLangs` in `syntaxHighlight`. This allows users to exclude languages from syntax highlighting. This is useful when rendering markdown that contains code blocks with specific languages which has integrations that depend on the language not being highlighted, such as mermaid diagrams.
+
+The default value for `excludeLangs` is `['math']` and is unchanged in this release.
```

Comment:
Noting that I will wait to review the changeset after we have docs, but if something is new, then how can its default value be unchanged? (compared to what?)

I'm confused by what this is trying to communicate, but don't want to miss something important! 😄

Reply:
Updated this.